04-conditional-logic/factors.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def get_a_positive_integer_from_user():
+    response = int(input("Enter a positive integer: "))
+
+    if not response:
+        get_a_positive_integer_from_user()
+
+    if response < 0:
+        print("You must provide a positive integer, try again...")
+        get_a_positive_integer_from_user()
+
+    return response
+
+def find_divusor_from_integer(integer):
```

Comment:
Typo: divisor instead of divusor.
Additionally, this function does something different than its title suggest. It prints all the divisors of the given integer rather than find a divisor.